ctx.go

```diff
@@ -833,6 +833,17 @@ func (c *Ctx) Params(key string, defaultValue ...string) string {
 	return defaultString("", defaultValue)
 }
 
+// Params is used to get all route parameters.
+// Using Params method to get params.
+func (c *Ctx) AllParams() map[string]string {
+	params := make(map[string]string, len(c.route.Params))
```

Comment:
```suggestion
func (c *Ctx) AllParams() map[string][]string {
	params := make(map[string][]string, len(c.route.Params))
```

Reply:
there is no list, always only one parameter value for one parameter key